### PR TITLE
BOM-64

### PR DESCRIPTION
**Pull request by Ayub-Khan**
https://github.com/edx/edx-oauth2-provider/pull/60
json is not able to serialize 'bytes' type object while simplejson can

### CODE REVIEW

File: provider/views.py [hunk 1]
```diff
@@ -580,9 +580,10 @@ def access_token_response(self, access_token, nonce=''):
         Returns a successful response after creating the access token
         as defined in :rfc:`5.1`.
         """
+        import simplejson as simple_json
         response_data = self.access_token_response_data(access_token, nonce=nonce)
         return HttpResponse(
-            json.dumps(response_data), content_type='application/json'
+            simple_json.dumps(response_data), content_type='application/json'
```
Comment (feanil) [inline provider/views.py]: Is response_data bytes in python 3 but string in python2?  In that case, can we wrap it in `six.text_type` to get a valid string in both cases?  If we do that we don't need to add a new library to the dependencies.

Comment (Ayub-Khan) [reply provider/views.py]: ok i will check that out. have we ran modernize on this repo.

Comment (feanil) [reply provider/views.py]: It looks like this repo is a fork of a 3rd party app @Ayub-Khan, have you looked at what the differences are between upstream an us?

**Comment by feanil** [comment]
@nasthagiri perhaps you have context.  Should we be trying to get off this fork and back onto upstream or should we just convert this repo for now with the hopes of not needing it in the future?

**Comment by clintonb** [comment]
We created this fork because upstream (https://github.com/caffeinehit/django-oauth2-provider) is abandoned. The last I check this is the most active fork of DOP. Most project maintainers in the Django community switched to django-oauth-toolkit (DOT).

**Comment by clintonb** [comment]
P.S. @caffeinehit may no longer exist. The homepage (http://caffeinehit.com/, which redirects to http://www.jabrikandfloon.com/) doesn't work. The only member of the org, @andy-a, has no public activity.

More links: https://www.crunchbase.com/organization/caffeinehit-ltd

**Comment by nasthagiri** [comment]
@clintonb Hi Stranger!  Thanks for the context.
@feanil We should spend as little time as possible on DOP.  We are about 6-developer-days away from moving edx.org off of DOP.  We need to resource completion of that effort since the hackathon last week.

**Comment by feanil** [comment]
In that case, I'm just gonna, run modernize on it and consider this code that edx owns.
Thanks fro the context @clintonb @nasthagiri

**Pull request closed without merging**